saferpayofficial.php

```diff
@@ -712,4 +714,24 @@ public function addFlash($msg, $type)
 
         return true;
     }
+
+    private function fixPaymentNotation(string $paymentNotation)
+    {
+        $fixedPaymentNotation = strtoupper($paymentNotation);
+        switch ($paymentNotation) {
+            case "AmericanExpress":
+                $fixedPaymentNotation = "AMEX";
+                break;
+            case "DinersClub":
+                $fixedPaymentNotation = "DINERS";
+                break;
+            case "Bonus Card":
```

Comment:
$paymentMethod['paymentMethod'] = str_replace(' ', '', $paymentMethod['paymentMethod']); would never reach this case.

Reply:
Yeah now I see, it is really annoying because from one API request for example returns American Express and in another AmericanExpress. Not sure how to adapt this place

Reply:
I guess I will remove white spaces between strings in all cases

Reply:
str_replace is used only in obtainPaymentMethodsNamesAsArray() but if we go to checkout were saved cards should be rendered in paymentOptions it is only  used obtainPaymentMethods() in this method str_replace is not used